packages/host/app/components/matrix/login.gts

```diff
@@ -15,58 +15,138 @@ import {
   BoxelInput,
   LoadingIndicator,
 } from '@cardstack/boxel-ui/components';
+import { BoxelIcon } from '@cardstack/boxel-ui/icons';
 
 import { isMatrixError } from '@cardstack/host/lib/matrix-utils';
 import type MatrixService from '@cardstack/host/services/matrix-service';
 
-export default class Login extends Component {
+interface Signature {
+  Args: {
+    onRegistration: () => void;
+  };
+}
+
+export default class Login extends Component<Signature> {
   <template>
-    <BoxelHeader @title='Login' @hasBackground={{true}} />
-    {{#if this.error}}
-      <div class='error' data-test-login-error>{{this.error}}</div>
-    {{/if}}
-    {{#if this.doLogin.isRunning}}
-      <LoadingIndicator />
-    {{else}}
-      <div class='login'>
-        <FieldContainer @label='Username:' @tag='label' class='login__field'>
+    <div class='login-form'>
+      <BoxelHeader @title='Boxel' @hasBackground={{false}} class='header'>
+        <:icon>
+          <BoxelIcon />
+        </:icon>
+      </BoxelHeader>
+      <div class='content'>
+        <span class='title'>Sign in to your Boxel Account</span>
+        <FieldContainer
+          @label='Email Address or Username'
```

Comment:
For now we are just using the username--and specifically the username without matrix embellishments. so if the matrix username is: `@fadhlan:localhost`, you would login with `fadhlan`. We should remove the message about the email, and add email login as part of a separate ticket (that will require using matrix 3rd party ID's which is a little different)

Reply:
I've filed a ticket https://linear.app/cardstack/issue/CS-6343/allow-users-sign-in-using-email.